server/models/comment.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+const {
+  Model
+} = require('sequelize');
+const bcrypt = require("bcrypt");
+module.exports = (sequelize, DataTypes) => {
+  class Comment extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
+      Comment.hasMany(models.Comment, {
+        foreignKey: "repliedToCommentId",
+        as: 'reply'
+      })
+
+      Comment.belongsTo(models.Comment, {
+        foreignKey: "repliedToCommentId"
+      })
+    }
+  };
+  Comment.init({
+    timeEntryId: DataTypes.INTEGER,
+    createdBy: DataTypes.INTEGER,
+    repliedToCommentId: DataTypes.INTEGER,
+    comment: DataTypes.TEXT,
+    isDeleted: DataTypes.ENUM('Y','N'),
```

Comment:
Please change isDeleted to boolean type and change corresponding methods according to new schema

Reply:
ok